page.codeberg.gwync.glerm.yml

```
@@ -0,0 +1,30 @@
+id: page.codeberg.gwync.glerm
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: glerm
+finish-args:
+  - --filesystem=home
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+modules:
+  - pypi-deps.json
+  - name: glerm
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 page.codeberg.gwync.glerm.metainfo.xml /app/share/metainfo/page.codeberg.gwync.glerm.metainfo.xml
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - install icon.png /app/share/icons/hicolor/256x256/apps/page.codeberg.gwync.glerm.png
+      - mkdir -p /app/share/applications/
+      - install -Dm 644 page.codeberg.gwync.glerm.desktop /app/share/applications/page.codeberg.gwync.glerm.desktop
```

```suggestion
      - install -Dm0644 icon.png /app/share/icons/hicolor/256x256/apps/page.codeberg.gwync.glerm.png
      - install -Dm 644 page.codeberg.gwync.glerm.desktop /app/share/applications/page.codeberg.gwync.glerm.desktop
```

install can create directories